Description: Correct spelling error in squeue binary
 This patch correct a spelling error in the squeue binary command.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>

---
Origin: other
Bug: http://bugs.schedmd.com/show_bug.cgi?id=1675
Last-Update: 2015-05-19

--- slurm-llnl-14.11.6.orig/src/squeue/opts.c
+++ slurm-llnl-14.11.6/src/squeue/opts.c
@@ -215,7 +215,7 @@ parse_command_line( int argc, char* argv
 			if (params.format == NULL) {
 				params.format_long = xstrdup(optarg);
 			} else {
-				error ("-O (--Format) is incompatable with -o "
+				error ("-O (--Format) is incompatible with -o "
 				       "(--format)");
 				exit(1);
 			}
